nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/actor/NodeReadsDataActor.scala

```diff
@@ -183,8 +168,8 @@ object MetricsDataActor {
                       enclosingConfig.getInt("upper-bound"))
   }
 
-  def props(basePath: String, nodeName: String, localCommitLogCoordinator: ActorRef): Props =
-    Props(new MetricsDataActor(basePath, nodeName, localCommitLogCoordinator))
+  def props(basePath: String, nodeName: String): Props =
+    Props(new NodeReadsDataActor(basePath, nodeName))
 
   case class AddRecordToLocation(db: String, namespace: String, bit: Bit, location: Location)
   case class DeleteRecordFromLocation(db: String, namespace: String, bit: Bit, location: Location)
```

Comment:
May we move writes/deletion related message to `NodeWritesDAtaActor` ?

Reply:
Good point. Thanks